docs/docs/recipes.md

```diff
@@ -189,3 +189,69 @@ since `0.7.4`).  Requires Mesos version `0.20.0` or later.
   "uris": ["http://downloads.mesosphere.com/misc/toggle.tgz"]
 }
 ```
+
+## Constraining task placement / static partitioning
+
+Usually, Mesos and Marathon collaboratively decide where to start your application tasks without your interference.
```

Comment:
I'd either leave out the word _collaboration_ and only write something like _Marathon decides where to start your tasks based on resource offers it receives from Mesos_. I know this is basically the same, but if you don't define what that _collaboration_ is, it might be confusing to mention that.

Apart from that: I like this little introduction before diving into the details. (It might make sense to enhance that, put it into a more general location and reference here later on – no need to do that now.)